app/scss/partials/_settings.scss

```diff
@@ -175,6 +175,11 @@
 			margin-top: 0;
 			margin-bottom: 11px;
 			text-transform: uppercase;
+			&.s-special {
+				display: inline-block;
+				margin-right: 8px;
+				max-width: 240px;
```

Comment:
@zarembsky Do we need a max-width here?

Reply:
We have to be sure that tooltip icon fits on the same line.